anndata/_io/merge.py

```diff
@@ -0,0 +1,647 @@
+from functools import singledispatch
+from .specs import read_elem, write_elem
+from .._core.merge import (
+    _resolve_dim,
+    StrategiesLiteral,
+    intersect_keys,
+    unify_dtypes,
+    merge_indices,
+    merge_dataframes,
+    resolve_merge_strategy,
+    gen_reindexer,
+    gen_inner_reindexers,
+    reduce,
+    concat_arrays,
+    MissingVal,
+    Reindexer,
+)
+from .._core.sparse_dataset import SparseDataset
+from .._core.file_backing import to_memory
+from ..experimental import read_dispatched
+import pandas as pd
+from typing import (
+    Any,
+    Callable,
+    Collection,
+    Optional,
+    Sequence,
+    Union,
+    Literal,
+    MutableMapping,
+    Mapping,
+)
+import typing
+from pathlib import Path
+import numpy as np
+
+from ..compat import ZarrGroup, ZarrArray, H5Group, H5Array
+
+from scipy.sparse import csr_matrix, csc_matrix
+
+MAX_LOAD_SIZE = 60_000_000
```

Comment:
Could this get a comment describing what the units are (e.g. elements, bytes) and why this value?

Reply:
It is currently elements; I was going to ask if this was ok or if I should use bytes instead. For now, I am writing it as elements.